python/dgl/distributed/sparse_emb.py

```diff
@@ -13,31 +13,32 @@ class SparseNodeEmbedding:
     ----------
     g : DistGraph
```

Comment:
It is weird to pass the graph to Embedding. Actually the embedding does not rely on any graph information.

Reply:
then what should we pass? unless we want to expose kvstore.